### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by marinsalinas**

### CODE REVIEW

**Comment by themantissa** [comment]
I will finish the review in the morning but please do not merge these till review is completed.

File: README.md [hunk 1]
```diff
@@ -2,10 +2,98 @@
...
+## Usage
+
+```go
+import "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+```
+
+## Authentication 
+
+The Atlas API uses [HTTP Digest Authentication](https://docs.atlas.mongodb.com/api/#api-authentication). Provide your Atlas PUBLIC_KEY as the username and RIVATE_KEY as the password as part of the HTTP request. See Programmatic API Keys docs for more detailed information: https://docs.atlas.mongodb.com/configure-api-access/#atlas-prog-api-key.
+
```
Comment (PacoDw) [inline README.md]: change RIVATE_KEY to PRIVATE_KEY

File: README.md [hunk 2]
```diff
@@ -2,10 +2,98 @@
...
+## Versioning
+Each version of the client is tagged and the version is updated accordingly.
+
+To see the list of past versions, run `git tag`.
+
+
+
+## Development and contribution
+
+Feel free to open a PR!. Our contribution guidelines are on WIP.
```
Comment (themantissa) [inline README.md]: Change a bit:
Feel free to open an Issue or PR!   Our contribution guidelines are a WIP but generally follow the offical [Terraform Guidelines](https://www.terraform.io/docs/extend/community/contributing.html).

Comment (marinsalinas) [reply README.md]: Changed!, TY!

**Pull request merged**